testing/client_app/helper/local_project_helper_utility.py

```
@@ -44,17 +46,134 @@ def __delete_directory(self, directory_name: str) -> bool:
+                    return False
+        return True
+
+    def move_file_to_untracked_folder(self, folder_name, project_title):
+        """ Move file to untracked folder
+
+        Args:
+            folder_name:
+            project_title:
+
+        Returns: returns the result of file move
+
+        """
+        user_credentials = ConfigurationManager.getInstance().get_user_credentials(LoginUser.User1)
+        username = user_credentials.user_name
+        home_dir = Path(GigantumConstants.HOME_DIRECTORY.value).expanduser()
+        if ConfigurationManager.getInstance().get_app_setting("default_server"):
+            default_server = ConfigurationManager.getInstance().get_app_setting("default_server")
+        else:
+            return False
+        file_path = home_dir / GigantumConstants.SERVERS_FOLDER.value / default_server / username / username\
+                    / GigantumConstants.PROJECTS_FOLDER.value / project_title / folder_name
+        source = file_path / GigantumConstants.DEFAULT_FILE_NAME.value
+        destination = file_path / GigantumConstants.UNTRACKED_FOLDER.value / GigantumConstants.DEFAULT_FILE_NAME.value
+        if file_path:
+            shutil.move(source, destination)
+            return True
+        return False
+
+    def verify_untracked_directory_is_empty(self, project_title) -> bool:
+        """ Verify untracked directories are empty or not
+
+        Returns: returns the result of verification
+
+        """
+        directories = ['code', 'input', 'output']
+        user_credentials = ConfigurationManager.getInstance().get_user_credentials(LoginUser.User1)
+        username = user_credentials.user_name
+        home_dir = Path(GigantumConstants.HOME_DIRECTORY.value).expanduser()
+        if ConfigurationManager.getInstance().get_app_setting("default_server"):
+            default_server = ConfigurationManager.getInstance().get_app_setting("default_server")
+        else:
+            return False
```

maybe raise an exception here instead of `false`? Really you're missing the setting to be able to check if the dir is empty. Returning false may result in unexpected test behavior.

---

Same with spots where this pattern is reused in this file (maybe worth moving into a function?)